src/main/java/shef/data/ForYou.java

```diff
@@ -12,6 +12,8 @@
 // See the License for the specific language governing permissions and
```

Comment:
Is this file supposed to be here (is it related to the rest of the PR)?

Reply:
I'm including it here to avoid potential nasty rebasing issues. Last week, I had conflicts that occurred because as I tried to rebase a branch to remove previous commits, git complained that the files didn't exist in previous commits in previous branches (or something like that, I'm not really sure what happened). I think including the empty files here is a workaround for that issue. ForYou.java will be the focus of a future Pull Request that depends on this one.